Makefile.create_config

```diff
@@ -24,4 +24,5 @@ src/ocamlbuild_config.ml: force
 	echo 'let so = "$(SO)"'; \
 	echo 'let ext_dll = "$(EXT_DLL)"'; \
 	echo 'let exe = "$(EXE)"'; \
+	echo 'let version = "$(shell cat VERSION)"'; \
```

Comment:
This would break on Windows. I know ocamlbuild doesn't work on Windows anyway, but we could avoid making the situation worse.